File: 2022/22.pl
#!/usr/bin/perl

use strict;
use warnings;

my $INPUT_FILE = $ARGV[0] if @ARGV>0 or die "No input file specified.\n";

sub day22_1 {
}

sub day22_2 {
}

open my $fh, "< ", $INPUT_FILE or die "$INPUT_FILE: $!";
chomp(my @data = <$fh>);
close ($fh);

day22_1();
day22_2();

File: 2023/22.py
#!/usr/bin/python3

import sys

if __name__ == "__main__":
    if (len(sys.argv)<2):
        print("Usage: 22.py file_to_use")
        exit(1)

    infile = open(sys.argv[1], "r")
    data = infile.readlines()
    infile.close()
